main.go

```diff
@@ -45,20 +46,23 @@ func init() {
 
 func main() {
 	var (
+		eventsAddr           string
 		metricsAddr          string
 		enableLeaderElection bool
 		concurrent           int
 		logJSON              bool
 	)
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8484", "The address the metric endpoint binds to.")
```

Comment:
source-controller has `:8080`, kustomize-controller `:8282`, this `:8484`?

Reply:
Yes, because I run them all locally.

Reply:
Please make sure helm-controller uses its own port.

Reply:
Wouldn't it be better for them to all be the same so things are predictable across components?

Reply:
All of them have the same port name and Prometheus uses the annotations we setup in the deployment, that port is not intended for a user to call into. We could use the same port and set the current ones in the makefile under run command if you want.

Reply:
> We could use the same port and set the current ones in the makefile under run command if you want.

I think having the same defaults for "standard flags" across components (even if they are not user facing) is the right thing to do.

Reply:
8080 for HTTP API and 9090 for Prom?

Reply:
kubebuilder automatically assigns `:8080` to Prom so I think the opposite.

Reply:
OK